pages/quizzes/html-quiz/html-quiz.html

```diff
@@ -0,0 +1,461 @@
+
+          <!-- Links -->
+          <a href="/index.html">
+            <h6 class="mr-6 cursor-pointer h6 text-hover-cyan-300 text-white">
+              Home
+            </h6>
+          </a>
+          <a href="/pages/quiz.html">
+            <h6 class="mr-6 cursor-pointer h6 text-hover-cyan-300 text-white">
+              Quizzes
+            </h6>
+          </a>
+          <!-- Links -->
+        </div>
+        <!-- header-left (logo, nav-links) -->
+
+        <!-- header middle (searchbar) -->
+        <form action="" class="search__bar__wrapper">
+          <input type="text" class="search__bar" placeholder="Search" />
+          <a href="#">
+            <i class="fas fa-search search__icon text-cyan-500"></i>
+          </a>
+        </form>
+        <!-- header middle (searchbar) -->
+
+        <!-- header right (nav-links, login/signup CTA) -->
+        <div class="flex justify-center align-items-center">
+          <!-- navigation links -->
+          <a href="/pages/wishlist-page/wishlist-page.html">
+            <span class="w-full h-full tooltip btn-icon mr-5">
+              <i
+                class="text-cyan-500 text-hover-cyan-300 fas fa-user text-3xl"
+              ></i>
+
+              <span
+                class="tooltip-text text-black text-lg tooltip-text-bottom bg-cyan-100"
+              >
+                Profile
+              </span>
+            </span>
+          </a>
+
+          <!-- login/signup CTA -->
+          <a
+            href="/pages/login-signup-page/login-page.html"
+            class="btn btn-square-solid btn-md text-white bg-cyan-600 btn-bg-hover-cyan-300 mr-7"
+          >
+            LOGIN
+          </a>
+
+          <!-- navigation links -->
+        </div>
+        <!-- header right (nav-links, login/signup CTA) -->
+      </div>
+    </header>
+    <!-- header -->
+
+    <div class="spacer-3rem"></div>
+    <div class="spacer-3rem"></div>
+    <div class="spacer-3rem"></div>
+    <!-- main section -->
+    <main style="top: 5rem; min-height: 100vh" class="relative w-full">
+      <div style="max-width: 980px; margin: auto" class="w-full">
+        <h3 class="text-4xl font-normal text-center text-black underline">
+          HTML MCQ
+        </h3>
+
+        <div class="spacer-3rem"></div>
+        <div class="spacer-3rem"></div>
+        <div class="flex justify-between align-items-center">
+          <a
+            href="/pages/quizzes/html-quiz/html-quiz-result.html"
+            class="btn shadow-md btn-square-solid bg-cyan-500 btn-bg-hover-cyan-900 btn-text-hover-white"
+            >Check Score</a
+          >
+
+          <div id="modal" style="z-index: 30" class="modal">
+            <div class="modal-content bg-white">
+              <div
+                class="modal-header flex justify-between align-items-center bg-grey-800 text-white p-5"
+              >
+                <h4 class="text-lg">Quiz Rules</h4>
+                <span
+                  onclick="closeModal()"
```

Comment:
Suggestion : It would be better if you add onClick functionality by fiving id to the element here and using the reference in JS